Bottom: 7e061e92870558c3d6dbb2decabc06eb07698e2f
Top:    7a1abfcf3372a3f8e59708892fe5812242e93bbd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:25:00 +0100

Refresh of draw-c-split-gathering-routine-2

---

diff --git a/src/draw.c b/src/draw.c
index 03073d4..58b60a1 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1817,7 +1817,7 @@ static void
 draw_element (ElementTypePtr element)
 {
   DrawElementPackage (element);
-  draw_element_name (Element);
+  draw_element_name (element);
   DrawElementPinsAndPads (element);
 }
